CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ All notable changes to this project are documented in this file.
 See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
 ## [Unreleased]
++ Engine
+    - Synchronize all files and directories for received snapshots [#4853](https://github.com/tikv/tikv/pull/4853)
```

Comment:
This is copy paste from 2.1.12 changelog.

Reply:
please change PR link